src/app/components/DevSiteRoutingMenu.jsx

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions  */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter, matchPath } from 'react-router-dom';
+import NavigationSideMenu from 'terra-navigation-side-menu';
+import RoutingStackDelegate from 'terra-navigation-layout/lib/RoutingStackDelegate';
+import SearchField from 'terra-search-field';
+import { FormattedMessage } from 'react-intl';
+
+const propTypes = {
+  /**
+   * The title to render within the DevSiteRoutingMenu's header.
+   */
+  title: PropTypes.string,
+  /**
+   * The array of routing shapes to be rendered as menu items within the DevSiteRoutingMenu.
+   */
+  menuItems: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
i know this was a copy + paste, but this propType is defined in `application-layout`. we can import that:
``` jsx
import { Utils } from 'terra-application-layout';
...
...
menuItems: Utils.propTypes.layoutConfigPropType
```

Reply:
👍 I originally had this, but the linter complained. I'll add it back with an eslint-disable